### PR TITLE
sys-fs/f2fs-tools-1.7.0: DEPS -> RDEPS

### PR DESCRIPTION
**Pull request by g-braeunlich**
f2fs-tools links against libselinux and util-linux

Package-Manager: portage-2.3.3

### CODE REVIEW

File: sys-fs/f2fs-tools/f2fs-tools-1.7.0.ebuild [hunk 1]
```diff
@@ -15,7 +13,7 @@ SLOT="0/1"
 KEYWORDS="~amd64 ~arm ~mips ~x86"
 IUSE=""
 
-DEPEND="
+RDEPEND="
 	sys-apps/util-linux
 	sys-libs/libselinux"
```
Comment (SoapGentoo) [inline sys-fs/f2fs-tools/f2fs-tools-1.7.0.ebuild]: the econf call seems weird. Can you try `--bindir="${EPREFIX}"/sbin` and chuck the `--prefix` and `--includedir` options and check whether you still get the same filesystem layouts.

File: sys-fs/f2fs-tools/f2fs-tools-1.7.0.ebuild [hunk 2]
```diff
@@ -15,7 +13,7 @@ SLOT="0/1"
...
```
Comment (SoapGentoo) [inline sys-fs/f2fs-tools/f2fs-tools-1.7.0.ebuild]: you're changing `RDEPEND` -> needs a revbump

Comment (SoapGentoo) [inline sys-fs/f2fs-tools/f2fs-tools-1.7.0.ebuild]: ALso, given that you said it links against the deps, you STILL need to incldue them in `DEPEND`

**Comment by blueness** [comment]
lgtm. pushed.

**Pull request closed without merging**